mapit_global/management/commands/mapit_global_update_countries.py

```diff
@@ -0,0 +1,129 @@
+from __future__ import print_function, unicode_literals
+
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+from django.utils.functional import cached_property
+from lxml import etree
+import requests
+
+from mapit.models import Area, Code, CodeType, Country, Generation
+
+
+class Command(BaseCommand):
+
+    OSM_COUNTRY_CODE_KEY = 'ISO3166-1'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--commit', action='store_true')
+        parser.add_argument('--generation', type=int)
+
+    def generation_kwargs(self):
```

Comment:
How come you didn't cached_property this, ooi? :)

Reply:
No particular reason, other than that it's going to be really fast anyway, and it'll be a tiny cost given how comparatively slow the point-in-polygon lookup is. I'll add it anyway, though :)

Reply:
It being a property does make uses of it tidier, too, I suppose